scripts/005-gcc-stage2.sh

```diff
@@ -59,7 +59,7 @@ rm -rf build-$TARGET-stage2 && mkdir build-$TARGET-stage2 && cd build-$TARGET-st
   --disable-libssp \
   --disable-multilib \
   --enable-threads=posix \
-  --disable-libstdcxx-pch \
+  # --disable-libstdcxx-pch \
```

Comment:
This will cause some issues, since the line below it will no longer be used. I suspect that's why the Mac build fails.

Reply:
Much better if totally remove it instead of commenting out for future usee